src/shared/firewall-util-nft.c

```diff
@@ -964,6 +967,48 @@ static int fw_nftables_add_masquerade_internal(
                                               source, source_prefixlen);
 }
 
+int nft_set_element_modify_in_addr_open(
+                FirewallContext **ctx,
+                bool add,
+                NFTSetContext *nft_set_context,
+                int af,
+                const union in_addr_union *address,
+                unsigned int prefixlen) {
+        const char *table, *set;
+        int r, nfproto;
+
+        assert(ctx);
+
+        if (!*ctx) {
+                r = fw_ctx_new_full(ctx, /* init_tables= */ false);
+                if (r < 0)
+                        return r;
+        }
+
+        ASSERT_PTR((*ctx)->nfnl);
```

Comment:
we usually use ASSERT_PTR() only when we want to use the return value. if you just want to check the value without caring for the return value, just use plain assert()

Reply:
I see, thanks for the explanation!